ivy/array/statistical.py

```diff
@@ -33,10 +33,84 @@ def mean(
         self: ivy.Array,
         /,
         *,
-        axis: Union[int, Tuple[int]] = None,
+        axis: Union[int, Sequence[int]] = None,
         keepdims: bool = False,
         out: Optional[ivy.Array] = None,
     ) -> ivy.Array:
+        """
+        ivy.Array instance method variant of ivy.mean. This method simply
+        wraps the function, and so the docstring for ivy.mean also applies
+        to this method with minimal changes.
+
+        **Special Cases**
+
+        Let ``N`` equal the number of elements over which to compute the arithmetic mean.
+        -   If ``N`` is ``0``, the arithmetic mean is ``NaN``.
+        -   If ``x_i`` is ``NaN``, the arithmetic mean is ``NaN`` (i.e., ``NaN`` values
+            propagate).
+
+        Parameters
+        ----------
+        self
+            input array. Should have a floating-point data type.
+        axis
+            axis or axes along which arithmetic means must be computed. By default, the mean
+            must be computed over the entire array. If a Sequence of integers, arithmetic means
+            must be computed over multiple axes. Default: ``None``.
+        keepdims
+            bool, if ``True``, the reduced axes (dimensions) must be included in the result
+            as singleton dimensions, and, accordingly, the result must be compatible with
+            the input array (see :ref:`broadcasting`). Otherwise, if ``False``, the reduced
+            axes (dimensions) must not be included in the result. Default: ``False``.
+        out
+            optional output array, for writing the result to.
+
+        Returns
+        -------
+        ret
+            array, if the arithmetic mean was computed over the entire array, a
+            zero-dimensional array containing the arithmetic mean; otherwise, a
+            non-zero-dimensional array containing the arithmetic means. The returned array
+            must have the same data type as ``x``.
+
+        Examples
+        --------
+        >>> x = ivy.array([3., 4., 5.])
+        >>> y = x.mean()
+        >>> print(y)
+        ivy.array(4.)
+
+        >>> x = ivy.array([3., 4., 5.])
```

Comment:
Could you vary the inputs a little bit? Repeating inputs are not helpful for others who will be reading your docstrings.
Also, pls add one sentence comment above each example to denote which kind of example it belongs to. For instance: `With :code:`ivy.Array` input:`. you can refer to the docstring section of [Deep Dive](https://lets-unify.ai/ivy/deep_dive/13_docstring_examples.html)

Reply:
Thanks @jiahanxie353  for your feedback. I will update accordingly